index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <nav>
+        <span>Glücklich am Deich</span>
+        <H1>Ferienhof Struve</H1>
+        <ul>
+            <li>Ferienwohnung</li>
+            <li>Ferienhaus</li>
+            <li>Unser Hof</li>
+            <li>Preise / Buchen</li>
+            <li>Umgebung</li>
+            <li>Kontakt</li>
+        </ul>
+    </nav>
+    <main>
+        <section>
```

Comment:
The indentation is not correct – so prettier does not work. Please check for that.